src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -102,7 +104,7 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
                 confirm: false,
             };
 
-            if (method === StripeElementType.CreditCard) {
+            if (method === StripeElementType.CreditCard && this._reusePaymentIntent) {
```

Comment:
Just double checking this is right.

`this._reusePaymentIntent === true` represent our new flow, as such we're turning this current behaviour _off_ 

(also removing `&& this._reusePaymentIntent` does not break any tests, so I think there's a scenario there that we haven't covered)

Reply:
the validation of this._reusePaymentIntent complemente the validation of the method, if you remove it test doesn't fail this because only ignore the parameter reusePaymentIntent received from the back and only validate the method